src/Catty/Storyboard+XIB/LaunchScreen.storyboard

```diff
@@ -1,11 +1,7 @@
 <?xml version="1.0" encoding="UTF-8"?>
-<document type="com.apple.InterfaceBuilder3.CocoaTouch.Storyboard.XIB" version="3.0" toolsVersion="11762" systemVersion="16C67" targetRuntime="iOS.CocoaTouch" propertyAccessControl="none" useAutolayout="YES" useTraitCollections="YES" colorMatched="YES" initialViewController="vwt-6C-EYF">
```

Comment:
Please revert your changes to this unintentionally changes file.

Reply:
reverted it.

Reply:
I am afraid now the file contains even more changes. Please revise your last commit. Thanks!

Reply:
now it should be resolved